Code/EditorPlugins/Assets/EditorPluginAssets/MaterialAsset/MaterialAsset.cpp

```diff
@@ -625,13 +625,13 @@ class ezVisualShaderErrorLog : public ezLogInterface
 
 ezStatus ezMaterialAssetDocument::InternalTransformAsset(const char* szTargetFile, const char* szOutputTag,
                                                          const ezPlatformProfile* pAssetProfile, const ezAssetFileHeader& AssetHeader,
-                                                         bool bTriggeredManually)
+                                                         ezBitflags<ezTransformFlags> transformFlags)
 {
   if (ezStringUtils::IsEqual(szOutputTag, ezMaterialAssetDocumentManager::s_szShaderOutputTag))
   {
     ezStatus ret = RecreateVisualShaderFile(AssetHeader);
 
-    if (bTriggeredManually)
+    if (transformFlags.IsSet(ezTransformFlags::TriggeredManually))
```

Comment:
What was this code path about?

Reply:
IIRC this has to do with recompiling shaders, ie. you need that when you modify the visual shader template files outside the editor and want to see the effect. In this case you probably want to use the Force flag instead, though.

Reply:
Thanks that explains it.